config/co2eq_parameters.js

```diff
@@ -14,7 +14,7 @@ exports.sourceOf = function(mode, zoneKey) {
 };
 
 exports.defaultExportIntensityOf = zoneKey =>
-  (co2eqParameters.fallbackZoneMixes[zoneKey] || {}).carbonIntensity;
+  (co2eqParameters.fallbackZoneMixes.zoneOverrides[zoneKey] || {}).carbonIntensity;
```

Comment:
for consistency, here you could also fallback on BP `defaults` (like you do for power origin).
Else we might have a power origin that uses defaults but a carbon intensity that is undefined (i.e. defaulting to country intensity instead of BP `defaults`)